controllers/provisioning_controller.go

```diff
@@ -398,7 +398,7 @@ func apiServerHost() (string, error) {
 	}
 	host := config.Host
 	if strings.Contains(apiServerURL.Host, ":") {
```

Comment:
I think it's OK to skip this check, net.SplitHostPort will do the right thing if there's no port